_includes/releases/v21.2/v21.2.8.md

```diff
@@ -0,0 +1,100 @@
+
+<h3 id="v21-2-8-operational-changes">Operational changes</h3>
+
+- The setting `kv.bulk_io_write.min_capacity_remaining_fraction` can now be set to cause bulk ingest operations like [`IMPORT`](../v21.2/import.html), [`RESTORE`](../v21.2/restore.html), or [`CREATE INDEX`](../v21.2/create-index.html) to fail rather than write to a node that is running out of disk space. [#78575][#78575]
+- Improved jobs system resilience to scheduled jobs that may lock up jobs/scheduled job table for long periods of time. Each schedule now has a limited amount of time to complete its execution. The timeout is controlled via the `jobs.scheduler.schedule_execution.timeout` [cluster setting](../v21.2/cluster-settings.html). [#77620][#77620]
+
+
+<h3 id="v21-2-8-command-line-changes">Command-line changes</h3>
+
+- - The `cockroach debug tsdump` command now allows viewing time-series data even in case of node failures by being re-run with the import filename set to `-`. [#77976][#77976]
+- Fixes a bug where running [`cockroach demo`](../v21.2/cockroach-demo.html) with the `--global` flag would not simulate latencies correctly when combined with the `--insecure` flag. [#78170][#78170]
+
+<h3 id="v21-2-8-db-console-changes">DB Console changes</h3>
+
+- Add **Hot Ranges** page and link to it on the sidebar. [#77594][#77594]
+- The `_status/nodes` endpoint is avaible to all users with the `VIEWACTIVITY` role option, not just Admins. In the **DB Console**, the **Nodes Overview**, and the **Node Reports** pages will now display unredacted information containing node hostnames and IP addresses for all users with the `VIEWACTIVITY` role option. Previously this was also gated for Admins only. [#78275][#78275]
+- Fixes a bug where a node in the `UNAVAILABLE` state will not have latency defined and cause the network page to crash. [#78627][#78627]
+
+<h3 id="v21-2-8-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused errors when attempting to create table statistics (with [`CREATE STATISTICS`](../v21.2/create-statistics.html) or [`ANALYZE`](../v21.2/explain-analyze.html)) for a table containing an index that indexed only virtual [`computed columns`](../v21.2/computed-columns.html). This bug has been present since version 21.1.0. [#77565][#77565]
+- Fixed a bug when adding a [`hash-sharded index`](../v21.2/hash-sharded-indexes.html) to a table watched by a changefeed. [#77739][#77739]
+- Fixed successive schema change backfills from skipping spans that were checkpointed by an initial backfill that was restarted. [#77829][#77829]
+- Attempting to run concurrent profiles works up to a concurrency limit of two. This will remove the occurrence of "profile id not found" errors while running upto two profiles concurrently. When a profile is not found, the error message has been updated to suggest remediation steps in order to unblock the user. [#77977][#77977]
```

Comment:
```suggestion
- Attempting to run concurrent profiles now works up to a concurrency limit of two. This will remove the occurrence of `profile id not found` errors while running up to two profiles concurrently. When a profile is not found, the error message has been updated to suggest troubleshooting steps. [#77977][#77977]
```